### PR TITLE
Remove secure cookie

### PR DESCRIPTION
**Pull request by ismaelhamed**
As stated in #4447, secure cookies never worked plus they're deprecated in the JVM anyway.

### CODE REVIEW

File: src/protobuf/WireFormats.proto [hunk 1]
```diff
@@ -82,7 +82,7 @@ message AkkaControlMessage {
 message AkkaHandshakeInfo {
 	AddressData origin = 1;
 	fixed64 uid = 2;
-	string cookie = 3;
+  optional string cookie = 3;
```
Comment (Aaronontheweb) [inline src/protobuf/WireFormats.proto]: I don't think this is needed - these fields are already optional by default.

**Review by Aaronontheweb — APPROVED** [review]
LGTM

File: src/protobuf/WireFormats.proto (hunk 1 above)
Comment (ismaelhamed) [reply src/protobuf/WireFormats.proto]: Everything in Proto3 is optional... I'll undo this.

Comment (Aaronontheweb) [reply src/protobuf/WireFormats.proto]: No worries - I took care of it

**Pull request merged**